app/lib/features/chat/widgets/room_avatar.dart

```diff
@@ -0,0 +1,160 @@
+import 'package:acter/common/providers/chat_providers.dart';
+import 'package:acter/features/home/providers/client_providers.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class RoomAvatar extends ConsumerWidget {
+  final String roomId;
+  final double avatarSize;
+
+  const RoomAvatar({
+    required this.roomId,
+    this.avatarSize = 36,
+    Key? key,
+  }) : super(key: key);
```

Comment:
Isn't this sufficient here:
```suggestion
    Key? super.key,
  });
```